util/base/src/test/java/jetbrains/jetpad/base/AsyncMatchers.java

```diff
@@ -38,7 +38,7 @@ protected boolean matchesSafely(Async<T> item, Description mismatchDescription)
               return false;
             }
           case FAILED:
-            mismatchDescription.appendText("failed");
+            mismatchDescription.appendText("failed with exception " + result.error);
```

Comment:
It would be nice to have tests illustrating these mismatch description messages in action.